.zuul.yaml

```diff
@@ -40,9 +40,19 @@
       A zuul job to validate the watcher operator and its service deployment.
     vars:
       run_tempest: false
+      # Donot use openstack services from meta content provider master
+      # job.
+      cifmw_update_containers: false
+    extra-vars:
+      # Override zuul meta content provider provided content_provider_dlrn_md5_hash
+      # var. As returned dlrn md5 hash comes from master release but job is using
+      # antelope content.
+      content_provider_dlrn_md5_hash: ''
```

Comment:
(trying to understand)... is an alternative to doing this (false update_containers and hash set to '') perhaps to have a -antelope content provider instead of a master one here? or we don't want/need -antelope content provider?

Reply:
ok thanks its a bit clearer now and i also read the commit message but theres a few things going on here. 

i'm still not clear why we need the content provider at all then in those cases we don't use the built containers from it.

Reply:
Note that we switched the default container images in the operator to use master since https://github.com/openstack-k8s-operators/watcher-operator/commit/5cf79efa1a41be3b95fac18ab7709177e447aa47 if that helps.